core/templates/dev/head/components/forms/schema_editors/SchemaBasedUnicodeEditorDirective.js

```diff
@@ -121,6 +123,65 @@ oppia.directive('schemaBasedUnicodeEditor', [
             return $sce.trustAsHtml(
               $filter('convertUnicodeWithParamsToHtml')($scope.localValue));
           };
+
+          var explorationLanguageCode = (
+            ExplorationLanguageCodeService.displayed);
+
+          var isSpeechRecognitionLanguagesupported = (
+            constants.SUPPORTED_SPEECH_RECOGNITION_LANGUAGES.filter(
```

Comment:
Correct me if I'm wrong but doesn't filter return another array?

Reply:
Yes, it does but I was checking if the returned array is empty or not. I have now shifted the check to this same line for avoiding confusion!